pkg/server/service_test.go

```diff
@@ -1,30 +1,35 @@
 package server
 
 import (
+	"github.com/moov-io/tr31"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 )
 
-func mockServiceInMemory() Service {
+func mockServiceInMock() Service {
+	repository := NewRepositoryInMemory(nil)
+	return NewMockService(repository)
+}
+func mockServiceInReal() Service {
 	repository := NewRepositoryInMemory(nil)
 	return NewService(repository)
 }
 func mockVaultAuthOne() Vault {
 	return Vault{
 		VaultAddress: "http://localhost:8200",
-		VaultToken:   "hvs.EqkXJUliZk0KUNII5lsydvGB",
+		VaultToken:   tr31.Local_vault_key,
 	}
 }
 func mockVaultAuthTwo() Vault {
 	return Vault{
-		VaultAddress: "https://https://portal.cloud.hashicorp.com:8200",
-		VaultToken:   "hvs.EqkXJUliZk0KUNII5lsydvGBC",
+		VaultAddress: "https://vault-cluster-public-vault-2d92a425.16ce2ded.z1.hashicorp.cloud:8200",
+		VaultToken:   tr31.Hashicorp_cloud_key,
```

Comment:
These can be read from env variables. `TR31_VAULT_TOKEN` and `TR31_VAULT_ADDRESS`.

Reply:
Fixed on new pull request